src/modules/app.ts

```diff
@@ -443,8 +443,14 @@ export class AppModule
       ],
       NFTCollection__factory,
     );
+    if (
+      metadata.feeRecipient !== undefined &&
+      metadata.feeRecipient !== address
```

Comment:
this should be protocol control / app address not the deploying module's address.